geomesa-ui/src/main/ui/bower.json

```diff
@@ -14,6 +14,7 @@
     "lodash": "~2.4.1",
     "moment": "~2.7.0",
     "moment-timezone": "~0.1.0",
-    "openlayers" : "https://github.com/openlayers/ol3/releases/download/v3.2.1/v3.2.1.zip"
```

Comment:
Should we include both openlayers 2 & 3? I'm guessing we added both b\c 2 has a helper function that 3 doesn't, but if there was another way to get around this I'd vote for that.